hack/pre-run-e2e.sh

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [[ "${HC_ACCESS_KEY}" = "" ]]; then
+  echo "HC_ACCESS_KEY can not be empty, please set it, e.g.: export HC_ACCESS_KEY=<your_ak>"
+  exit 1
+fi
+
+if [[ "${HC_SECRET_KEY}" = "" ]]; then
+  echo "HC_SECRET_KEY can not be empty, please set it, e.g.: export HC_SECRET_KEY=<your_sk>"
+  exit 1
+fi
+
+echo -e "\nBuild images"
+export REGISTRY_SERVER_ADDRESS=swr.ap-southeast-1.myhuaweicloud.com VERSION=latest
+make images
```

Comment:
right, we have to build the image before prepare the environment.
But, I can't see where you `load` the image to the node, I'm afraid the container can't load the image. Are you testing it on a localhost cluster or a real cluster, like build by kubeadm?

Reply:
Currently we are running the test environment built by minikube, which may not work properly under the k8s cluster. I will fix it in the next PR.

Reply:
I suppose you can remove the 3 scripts from this patch, and send another separated PR for this.
The E2E part is LGTM and ready to go.

Reply:
ok, I will send these scripts in another PR